src/polymer/js-utils.ts

```diff
@@ -146,30 +146,26 @@ export function toScannedMethod(
 export function getOrInferPrivacy(
     name: string,
     annotation: jsdoc.Annotation|undefined,
-    privateUnlessDocumented: boolean): Privacy {
+    inferFromName = true,
```

Comment:
Well, I don't know if we want to adopt the underscore-naming rules to all kinds of javascript entities. Does it make as much sense for element classes as it does for methods, for example? Pretty sure you'll find many people would be offended by the idea of prefixing class names with underscores.